skopt/space/space.py

```diff
@@ -188,6 +200,10 @@ def __init__(self, low, high, prior="uniform", transform=None, name=None):
               and upper bounds.
             - If `"log-uniform"`, points are sampled uniformly between
               `log10(lower)` and `log10(upper)`.`
```

Comment:
Should probably be changed to something akin `log(lower)` and `log(upper)`, where `log` has base `base`.' 

That detail aside, it may be overkill to introduce anything other than `Log10` and `Log2` (and *maybe* the natural log). On the other hand, with `LogN` one only needs one class, and can get rid of the `Log10` class. ... But I'll leave it to the regular contributors to discuss that.